### PR TITLE
add series record

### LINKED ISSUE #255: Volumes and series structure and handling
- [ ] Design the solution of storing the volumes with taken into an account migration rules (check cds-dojson repository)
- [ ] Implement the solution with schema, and handling the volumes in the backend

### LINKED ISSUE #347: Volumes and series - backend
- [x] implement data model, update schemas, data validation
- [x] implement necessary resolvers

### PR DESCRIPTION
**Pull request by equadon**
Closes #347

### CODE REVIEW

**Comment by equadon** [comment]
Right now `Document` and `Series` have many duplicate fields. Do you think it would be a good idea to store the common fields in a separate schema and include it in the document and series schema?

**Comment by ntarocco** [comment]
> Right now `Document` and `MultiVolume` have many duplicate fields. Do you think it would be a good idea to store the common fields in a separate schema and include it in the document and multi volume schema?
> 
> I'm also wondering if we actually need fields like `languages`, `authors`, keywords, etc when that information could be gathered by going through the child documents inside the `MultiVolume`. What do you think?

We can gather all these questions and ask to librarians. I think that we should find the most complete MultiVolume in current CDS and see what fields are different.
About the schema split, well, it is a bit tricky with Invenio I think... can we brainstorm this together?

File: invenio_app_ils/mappings/v6/documents/document-v1.0.0.json [hunk 1]
```diff
@@ -39,6 +39,26 @@
             }
           }
         },
+        "series": {
```
Comment (kpsherva) [inline invenio_app_ils/mappings/v6/documents/document-v1.0.0.json:42]: the `series_pids` are missing from the mapping

File: invenio_app_ils/mappings/v6/series/series-v1.0.0.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
+{
+  "mappings": {
```
Comment (kpsherva) [inline invenio_app_ils/mappings/v6/series/series-v1.0.0.json:2]: we should ask Anne which fields might be in series

File: invenio_app_ils/mappings/v6/documents/document-v1.0.0.json (hunk 1 above)
Comment (equadon) [reply invenio_app_ils/mappings/v6/documents/document-v1.0.0.json:42]: Do you think it's necessary to have it when the `series` object has the same information?

**Review by kpsherva — APPROVED** [review]

**Review by vlad-bm — APPROVED** [review]

**Pull request merged**